src/azul/indexer/notify_service.py

```diff
@@ -0,0 +1,54 @@
+import json
+import logging
+
+from azul import (
+    JSON,
+    config,
+)
+from azul.deployment import (
+    aws,
+)
+from azul.strings import (
+    trunc_ellipses,
+)
+
+log = logging.getLogger(__name__)
+
+
+class AzulEmailNotificationService:
+
+    def notify_group(self, subject: str, message: str) -> None:
+        log.info('Notifying group of event %r', trunc_ellipses(message, 256))
+        try:
+            # Ensure readable formatting …
+            body = json.loads(message)
+        except json.decoder.JSONDecodeError:
+            log.warning('Not a JSON serializable event, sending as received.')
+            body = message
+        else:
+            # … by deserializing with indent. However, it must happen as a
+            # second step (not in .loads) to avoid raising a JSONDecodeError
+            body = json.dumps(body, indent=4)
```

Comment:
"Ensure" is not an accurate word choice it's possible that we won't be able to format the message. I also think the second half of the comment is more verbose than it needs to be given the simplicity of the code it describes.

```suggestion
        # Try to improve readability by adding indent
        try:
            body = json.loads(message)
        except json.decoder.JSONDecodeError:
            log.warning('Not a JSON serializable event, sending as received.')
            body = message
        else:
            body = json.dumps(body, indent=4)
```